PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Description
+Please explain the changes you made here.
+
+### Testing done
```

Comment:
Should we include updating documentation to this list?

There are also a number of comments on API template that may also make sense to include here:
```
- is there a feature flag for this change, and if so, does the devops team need to be made aware of varying configuration for each environment?
...
- are there dashboards and associated instrumentation in place for this change? Have alerts been configured?
- Which alerts would indicate an issue with this functionality?
```

Similar to the alert comment above, error logging in Sentry might be helpful to include here.


We also don't have an automated solution in place for cross-browser testing, perhaps a section for `#### How should this be manually tested?` could help address that kind of gap.

Reply:
@jkassemi  That was a oversight on my part. When I created this one, I didn't know there was an existing pull request template in the vets-api repository. I've updated this one to match the one in vets-api.

@cehsu I've updated the Testing done and Testing Planned html comments to mention manual testing and cross browser testing. And also included the Acceptance Criteria section that is also in the vets-api pull request template.